### DISCUSSION TITLE
Does babel support type modifiers on import name?

### DISCUSSION THREAD

**Discussion by SyntheticGoop** [babel/babel#14297] [Q&A] [answered]
Hi, I've had an issue with nextjs not compiling when using babel: https://github.com/vercel/next.js/discussions/34666#discussion-3893498.

The following currently fails to build with Syntax error: Unexpected token, expected ","

```tsx
import { type T } from "..."

export default () => <div></div>
```

**Comment by nicolo-ribaudo** [comment, marked as answer]
It does: https://babeljs.io/repl#?browsers=Chrome%2090&build=&builtIns=false&corejs=3.6&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAbzjAnmApnAKnAvnAMyghDgCIA6KsgKBvQA9JY4ATdAgQwFcAbeABQBKOAF4AfHAA8rYADdxUgPSyFQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=env%2Creact%2Cstage-2%2Ctypescript&prettier=false&targets=&version=7.17.6&externalPlugins=&assumptions=%7B%7D

Probably you are using an old Babel version.
